code/modules/client/preferences_toggles.dm

```diff
@@ -71,19 +71,23 @@
 	feedback_add_details("admin_verb","TP") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
 /client/verb/toggletitlemusic()
-	set name = "Hear/Silence LobbyMusic"
+	set name = "Lobby Music Volume"
 	set category = "Preferences.Sound"
-	set desc = "Toggles hearing the GameLobby music"
-	prefs.toggles ^= SOUND_LOBBY
-	prefs.save_preferences()
-	if(prefs.toggles & SOUND_LOBBY)
+	set desc = "Adjusts the Game Lobby music volume"
+
+	var/lobby_music_vol_new = input(src, "Choose lobby music volume, 0 to mute", "Global Preference", prefs.lobby_music_vol) as num|null
+	if(isnull(lobby_music_vol_new))
+		lobby_music_vol_new = 0
+	if(!prefs.lobby_music_vol && lobby_music_vol_new)
 		to_chat(src, "You will now hear music in the game lobby.")
```

Comment:
print the new volume as well

Reply:
i think it's fine - you can audibly hear the volume changing